### DISCUSSION TITLE
Globe.dev instead of AWS ?

### DISCUSSION THREAD

**Discussion by dongri727** [serverpod/serverpod#1658] [Q&A] [answered]
Is it possible to deploy ServerpodProject to Globe?
https://globe.dev/docs

**Comment by vlidholt** [comment, marked as answer]
Currently no, this is because Globe has very limited support for databases (Serverpod requires Postgres) and it cannot handle multiple ports (by default Serverpod uses three ports, one for the API, one for Insights, and one for the web server).

However, we are working on "Serverpod mini", which isn't using the database. It's likely that can be run on Globe.
https://github.com/serverpod/serverpod/issues/1725
